tox.ini

```diff
@@ -5,18 +5,21 @@ envlist =
 
 [testenv]
 passenv=HOME
-deps =
-    pytest
-    pytest-cov
-    pytest-datadir
 sitepackages = False
+deps = -r{toxinidir}/requirements.txt
+       -r{toxinidir}/test-requirements.txt
+       -r{toxinidir}/dist-requirements.txt
 commands =
     pytest --cov-report=term-missing --cov=toolchest tests
 
 
 [testenv:flake8]
 passenv=HOME
 sitepackages = False
-deps = flake8
 commands =
     flake8 --ignore=E501,W504,E741 setup.py docs toolchest tests
+
+[testenv:twine]
```

Comment:
Wonder if we should add a CI job using this, too? Not necessarily with this PR.

Reply:
Agree on adding it to CI. I don't mind adding it now, but I think if I leave it for a separate PR, we might be able to use this one to make sure mergify is now working correctly.

Reply:
Makes sense to me!